src/main/java/com/academy/task/repository/EmailRepositoryImpl.java

```diff
@@ -0,0 +1,160 @@
+package com.academy.task.repository;
+
+import com.academy.task.model.Email;
+import com.academy.task.util.ConnectionProperties;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Repository;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class EmailRepositoryImpl implements EmailRepository {
+
+    ConnectionProperties properties;
+
+    @SneakyThrows
+    private Connection getDBConnection() {
+        Class.forName(properties.getDriver());
+
+        return DriverManager.getConnection(properties.getURL(), properties.getUSER(), properties.getPASS());
+    }
+
+    @Override
+    @SneakyThrows
+    public void add(Email email) {
+        PreparedStatement statement = getDBConnection().prepareStatement("INSERT INTO email(id, recipient, subject, body, deliver_date) VALUES (?, ?, ?, ?, ?)");
```

Comment:
Hibernate implementation should be implemented with usage of JPA.
Here you create SQL query manually. It should be automatically generated by Hibernate.